.github/ISSUE_TEMPLATE/bug_dev_edition.yaml

```
@@ -0,0 +1,78 @@
+name: "🐞 Bug Report - Dev Edition"
+description: "Report a bug if something isn't working as expected in the Elementor development versions"
+title: "🐞 Bug report: "
+labels: [bug,dev-edition]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## BEFORE POSTING YOUR ISSUE
+
+        - Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!
+
+        - If you have questions or need general support,  Please use:  https://wordpress.org/support/plugin/elementor
```

```suggestion
        - If you have questions or need general support, please use:  https://wordpress.org/support/plugin/elementor
```

Plus same as above regarding other things

---

Done, also in `custom.yaml`.